gen/generator.go

```diff
@@ -956,6 +1218,26 @@ func (m *Model) SetResourceNameAsDescription(classPkgName string, definitions De
 	m.ResourceNameAsDescription = GetResourceNameAsDescription(GetResourceName(classPkgName, definitions), definitions)
 }
 
+func (m *Model) SetApicType(classDetails interface{}) {
+	platform := make(map[string][]interface{})
+	if platformFlavors, ok := classDetails.(map[string]interface{})["platformFlavors"].([]interface{}); ok {
+		platform["apicType"] = platformFlavors
+		m.PlatformFlavors = platform
+	}
+}
+
+func (m *Model) SetClassExclude() {
+	if classDetails, ok := m.Definitions.Classes[m.PkgName]; ok {
+		for key, value := range classDetails.(map[interface{}]interface{}) {
+			if key.(string) == "exclude" {
+				m.Exclude = value.(bool)
+			} else {
+				m.Exclude = false
+			}
```

Comment:
this should be removed I think, else if it is not the last key it would be false, same applies for the set_class_include function

Reply:
Done!